packages/template-retail-react-app/app/translations/en-US.json

```diff
@@ -299,7 +299,7 @@
     "defaultMessage": "Terms & Conditions"
   },
   "checkout_footer.message.copyright": {
-    "defaultMessage": "2021 Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed."
+    "defaultMessage": "Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed."
```

Comment:
Re: the translations, we'll need to update all of the other locales too. And then re-compile them.

Reply:
Thanks. Fixed